src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -142,9 +140,18 @@
                             }
                             else
                             {
-                                <button type="submit" class="btn btn-sm shadow-sm btn-success ml-2">
-                                    <i class="far fa-thumbs-up" aria-hidden="true"></i>&nbsp;&nbsp;Approve
-                                </button>
+                                @if (Model.approvers.Contains(User.GetGitHubLogin()))
+                                {
+                                    <button type="submit" class="btn btn-sm shadow-sm btn-success ml-2">
+                                        <i class="far fa-thumbs-up" aria-hidden="true"></i>&nbsp;&nbsp;Approve
+                                    </button>
+                                }
+                                else
+                                {
+                                    <button type="submit" class="btn btn-sm shadow-sm btn-success ml-2" disabled>
+                                        <i class="far fa-thumbs-up" aria-hidden="true"></i>&nbsp;&nbsp;Cannot Approve
+                                    </button>
+                                }

```

Comment:
I don't think we need to show the button when it can't be approved. @JonathanGiles I would like to know your suggestion on this from a reviewer perspective.

Reply:
Button changed from 'disabled' to 'hidden'